DataBinFE/src/pages/Dashboard.tsx

```diff
@@ -21,14 +21,14 @@ export const Dashboard = () => {
   }, []);
 
   return (
-    <div className="flex h-screen w-screen flex-col dashboard-container" >
+    <div className="flex h-screen w-screen flex-col dashboard-container">
       {loading ? (
         <div className="flex items-center justify-center h-full">
           <ProgressSpinner />
         </div>
       ) : (
         <>
-          <div className="flex flex-col" style={{overflowY:'hidden'}}>
+          <div className="flex flex-col" style={{ overflowY: 'hidden', overflowX: 'hidden' }}>
```

Comment:
this is incorrect implementation. If he hide the overflow the contents will go missing. Check the content of right side of the page when you hover over the sidebar